systemd/systemd.go

```diff
@@ -638,6 +640,14 @@ func (s *systemd) Restart(serviceName string, timeout time.Duration) error {
 	return s.Start(serviceName)
 }
 
+func (s *systemd) RestartAll(serviceName string) error {
+	if s.mode == GlobalUserMode {
+		panic("cannot call restart with GlobalUserMode")
+	}
+	_, err := s.systemctl("restart", serviceName, "--all")
```

Comment:
I looked at systemd code, even though the systemctl manpage does not mention `--all` for restart, the implementation of `restart` is the same as of start `start` and should do the right thing. Quick testing in a VM confirms that.